package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "headquarters-node",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
Stuff changed internally, it's the same on the outside. I found something some interesting [controversies](https://github.com/mojombo/semver/issues/156) about semver. Anyway, this package doesn't care if you're trying to register with an invalid email, it changes nothing. The way I see it, the problem is that the package is coupled to our installation of the headquarters. We could change that and make the developers specify the api url, that way we won't have to bump a new version next time we change name, or someone else happens to start use headquarters.